app/src/main/kotlin/com/wire/android/notification/CallNotificationManager.kt

```diff
@@ -106,7 +158,7 @@ class CallNotificationManager @Inject constructor(private val context: Context)
         return NotificationCompat.Builder(context, channelId)
             .setContentTitle(callName)
             .setContentText(context.getString(R.string.notification_ongoing_call_content))
-            .setPriority(NotificationCompat.PRIORITY_HIGH)
+            .setPriority(NotificationCompat.PRIORITY_DEFAULT)
```

Comment:
We don't need this ongoing call notification to pop up on screen every time we put the app into the background as the user already is aware that the call is ongoing - it can be a little bit silent one so it shows but doesn't disturb the screen. Do you have different opinion on that?

Reply:
Makes sense👍